diff --git a/dist/bindings/utils/cloneLlamaCppRepo.js b/dist/bindings/utils/cloneLlamaCppRepo.js
index c946cc229eb4f3d39ba2c43e55ff553e854e08b6..a9a1a42bc1eddba0cba340d828406a99ca3008c0 100644
--- a/dist/bindings/utils/cloneLlamaCppRepo.js
+++ b/dist/bindings/utils/cloneLlamaCppRepo.js
@@ -18,7 +18,7 @@ export async function cloneLlamaCppRepo(githubOwner, githubRepo, tag, useBundles
     const remoteGitUrl = `https://github.com/${githubOwner}/${githubRepo}.git`;
     async function withGitCloneProgress(cloneName, callback) {
         if (!progressLogs)
-            return await callback(simpleGit({}));
+            return await callback(simpleGit({ unsafe: { allowUnsafePack: true } }));
         const repoText = `${githubOwner}/${githubRepo} (${cloneName})`;
         let lastProgress = 0;
         let stages = 1;
@@ -29,6 +29,7 @@ export async function cloneLlamaCppRepo(githubOwner, githubRepo, tag, useBundles
             progressFractionDigits: false
         }, async (progressUpdater) => {
             const gitWithCloneProgress = simpleGit({
+                unsafe: { allowUnsafePack: true },
                 progress({ progress }) {
                     const currentProgress = progress / 100;
                     if (currentProgress < lastProgress)
